Source/Core/VideoBackends/Vulkan/SwapChain.cpp

```diff
@@ -346,6 +345,17 @@ bool SwapChain::CreateSwapChain()
                                               m_present_mode,
                                               VK_TRUE,
                                               old_swap_chain};
+  if (g_vulkan_context->GetGraphicsQueueFamilyIndex() !=
+      g_vulkan_context->GetPresentQueueFamilyIndex())
+  {
+    std::array<uint32_t, 2> indices = {{
+        g_vulkan_context->GetGraphicsQueueFamilyIndex(),
+        g_vulkan_context->GetPresentQueueFamilyIndex(),
+    }};
+    swap_chain_info.imageSharingMode = VK_SHARING_MODE_CONCURRENT;
+    swap_chain_info.queueFamilyIndexCount = 2;
+    swap_chain_info.pQueueFamilyIndices = indices.data();
```

Comment:
I think that `indices` is going out of scope here, so the pointer from `indices.data()` is no longer valid by the time it gets to `vkCreateSwapchainKHR`.

Reply:
Yep. Probably the easiest solution is to move the array out of the if block.

Reply:
Whoops, fixed.